src/app/features/rpc-transaction-request/rpc-transaction-request.layout.tsx

```diff
@@ -2,41 +2,41 @@ import { Outlet } from 'react-router-dom';
 
 import { Box } from 'leather-styles/jsx';
 
-import type { Money } from '@leather.io/models';
+import { type RpcMethodNames, stxTransferStx } from '@leather.io/rpc';
 import { Approver } from '@leather.io/ui';
-import { baseCurrencyAmountInQuote, i18nFormatCurrency } from '@leather.io/utils';
 
-import { closeWindow } from '@shared/utils';
+import { analytics } from '@shared/utils/analytics';
 
 import type { HasChildren } from '@app/common/has-children';
 import { BackgroundOverlay } from '@app/components/loading-overlay';
-import { getTransactionActions } from '@app/components/rpc-transaction-request/get-transaction-actions';
-import { TransactionActionsTitle } from '@app/components/rpc-transaction-request/transaction-actions-title';
-import { TransactionError } from '@app/components/rpc-transaction-request/transaction-error';
 import { TransactionHeader } from '@app/components/rpc-transaction-request/transaction-header';
 import { TransactionWrapper } from '@app/components/rpc-transaction-request/transaction-wrapper';
-import { useFeeEditorContext } from '@app/features/fee-editor/fee-editor.context';
-import { useRpcTransactionRequestContext } from '@app/features/rpc-transaction-request/rpc-transaction-request.context';
+
+import { useRpcTransactionRequest } from './use-rpc-transaction-request';
+
+function getHelpLinkUrl(method: RpcMethodNames) {
+  switch (method) {
+    case stxTransferStx.method:
+      return 'https://leather.io/guides/send-stacks-from-app';
+    default:
+      return 'https://leather.io/guides/connect-dapps';
+  }
+}
```

Comment:
IMO this should be passed in to avoid a "do everything" fn for all possible methods.  Describing this link should be part of the implementation of each method

Suggest passing in as prop instead